src/mlpack/methods/ann/activation_functions/elliot_function.hpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file elliot_function.hpp
+ * @author Bisakh Mondal
+ *
+ * Definition and implementation of the Elliot Activation function as described 
+ * by D.L. Elliott.
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @techreport{Elliott1993,
+ *   title = {A better activation function for artificial neural networks},
+ *   author = {Elliott, David L},
+ *   url = {https://drum.lib.umd.edu/bitstream/handle/1903/5355/TR\_93-8.pdf}
```

Comment:
The link doesn't work for me, maybe we should use: https://drum.lib.umd.edu/handle/1903/5355 ?

Reply:
Thanks, @zoq for pointing out. Actually it's my fault here -> `TR\_93-8.pdf`. Unlike latex where for printing `_` we generally pass `\_`, but as I see that's not the case with doxygen. 
![Screenshot from 2020-05-02 08-22-13](https://user-images.githubusercontent.com/41498427/80853520-33f1a180-8c4f-11ea-94b8-ab7d6b360953.png)

 this [link](https://drum.lib.umd.edu/bitstream/handle/1903/5355/TR_93-8.pdf) will work fine.